setup.py

```diff
@@ -75,7 +75,7 @@ def run_tests(self):
 
 about = {}
 here = os.path.abspath(os.path.dirname(__file__))
-with open(os.path.join(here, "requests", "__version__.py"), "r", "utf-8") as f:
+with open(os.path.join(here, "requests", "__version__.py"), "r", encoding="utf-8") as f:
     exec(f.read(), about)
 
 with open("README.md", "r", "utf-8") as f:
```

Comment:
@lucas-ccc - L81 may also result in the same error.

```suggestion
with open("README.md", "r", encoding="utf-8") as f:
```